WordPress/Classes/ViewRelated/Notifications/Comment Moderation/CommentModerationState.swift

```diff
@@ -1,6 +1,6 @@
-enum CommentModerationState: CaseIterable {
+enum CommentModerationState: Equatable {
     case pending
-    case approved
-    case liked
+    case approved(liked: Bool)
+    case spam
     case trash
```

Comment:
1. I removed the liked case as it seems more like a substate of the approved state rather than a distinct state within the moderation view.
2. I added the `spam` state.